apps/docs/app/features/portable-text/PortableText.tsx

```diff
@@ -266,6 +267,19 @@ const components: Partial<PortableTextReactComponents> = {
       </Box>
     ),
     imports: ({ value }) => <CodeBlock code={value.reactImport} mt={3} />,
+    tipsPanel: ({ value }) => (
+      <Box backgroundColor="alias.mint" mt={3} p={4} borderRadius="md">
+        <Flex gap={1} alignItems="end">
+          <FavouriteOutline30Icon />
+          <Heading as="h3" textStyle="sm" fontWeight="bold">
+            {value.title}
+          </Heading>
+        </Flex>
+        <Box __css={{ " > p:first-of-type": { mt: 1 } }}>
```

Comment:
Standard-avstand er 2 steg på skalaen. Det blir for mye i denne konteksten. Så da bruker jeg denne litt stygge workarounden.